tests/performance/test_cluster/test_cluster_performance/test_cluster_performance.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+from datetime import datetime
+from os import listdir
+from os.path import join, dirname, realpath
+
+import pytest
+from yaml import safe_load
+
+from wazuh_testing.tools.performance.csv_parser import ClusterCSVTasksParser, ClusterCSVResourcesParser, ClusterEnvInfo
+
+test_data_path = join(dirname(realpath(__file__)), 'data')
+configurations = {file.replace('_thresholds.yaml', ''): safe_load(open(join(test_data_path, file))) for file in
+                  listdir(test_data_path)}
+date_format = '%Y/%m/%d %H:%M:%S'
+exceeded_thresholds = []
+
+
+# Fixtures
+@pytest.fixture()
+def artifacts_path(pytestconfig):
+    return pytestconfig.getoption("artifacts_path")
+
+
+@pytest.fixture()
+def n_workers(pytestconfig):
+    return pytestconfig.getoption("n_workers")
+
+
+@pytest.fixture()
+def n_agents(pytestconfig):
+    return pytestconfig.getoption("n_agents")
+
+
+# Functions
+def get_datetime_diff(phase_datetimes):
+    """Calculate the difference between two datetimes.
+
+    Args:
+        phase_datetimes (list): List containing start and end datetimes.
+    """
+    return datetime.strptime(phase_datetimes[1], date_format) - datetime.strptime(phase_datetimes[0], date_format)
+
+
```

Comment:
`get_datetime_diff` should be part of common Framework functions and no local test function

Reply:
To be done at https://github.com/wazuh/wazuh-qa/issues/2287